consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/support/TestContextBuilder.java

```diff
@@ -254,7 +258,7 @@ private static Block createGenesisBlock(final Set<Address> validators) {
     final BftExtraData extraData =
         new BftExtraData(
             Bytes.wrap(new byte[32]), Collections.emptyList(), Optional.empty(), 0, validators);
-    headerTestFixture.extraData(extraData.encode());
+    headerTestFixture.extraData(new QbftExtraDataEncoder().encode(extraData));
```

Comment:
nit: can the class member `bftExtraDataEncoder` use here?

Reply:
it can if I make it static, which I can. done